docs/release/helm-chart.md

```diff
@@ -1,20 +1,25 @@
-# Helm charts release 
+# Helm charts release
 
-We host all Helm charts on [kuberay-helm](https://github.com/ray-project/kuberay-helm). This document describes the process for release managers to release Helm charts.
+We host all Helm charts on [kuberay-helm](https://github.com/ray-project/kuberay-helm).
+This document describes the process for release managers to release Helm charts.
 
 ## The end-to-end workflow
 ### Step 1: Update version in Chart.yaml
```

Comment:
Maybe we need to add a new step to check `image.tag` in `values.yaml`. We update them at https://github.com/ray-project/kuberay/pull/784, but it is easy to forget.

Reply:
Oh, hm. Right now, I'm including that step in release.md.
I think it's easiest to update all of the versions in a single PR -- it's pretty straightforward to do by grepping for "nightly" and the "0.3.0".

Reply:
Let me add a blurb here too.